src/ducks/Studio/url.js

```diff
@@ -8,7 +8,20 @@ const { trendPositionHistory } = Events
 const COMPARE_CONNECTOR = '-CC-'
 const getMetricsKeys = metrics => metrics.map(({ key }) => key)
 const toArray = keys => (typeof keys === 'string' ? [keys] : keys)
-const convertKeyToMetric = (key, dict) => dict[key] || compatabilityMap[key]
+const convertKeyToMetric = (key, dict) =>
+  dict[key] || compatabilityMap[key] || getFromSubmetrics(key, dict)
+
+const getFromSubmetrics = (key, metrics) => {
```

Comment:
For a readability reasons better to use function declaration syntax if it's not a single statement